secator/utils.py

```diff
@@ -439,3 +439,26 @@ def print_version():
 	console.print(f'[bold gold3]Lib folder[/]: {LIB_FOLDER}')
 	if status == 'outdated':
 		console.print('[bold red]secator is outdated, run "secator update" to install the latest version.')
+
+
+def extract_root_domain_from_domain(domain):
```

Comment:
This definitely requires a unittest to check if it catches all the cases (weirdly formatted domains, multi-level subdomains, domains with special characters in the name, etc...